pkg/apicloudwatch/client.go

```diff
@@ -13,7 +12,7 @@ import (
 )
 
 type CloudWatchClient interface {
-	ListMetrics(ctx context.Context, namespace string, metric *config.Metric) (*cloudwatch.ListMetricsOutput, error)
+	ListMetrics(ctx context.Context, namespace string, metric *config.Metric, recentlyActiveOnly bool) (*cloudwatch.ListMetricsOutput, error)
```

Comment:
@PerGon if you rebase you'll have a conflict here as the signature of this method has changed on `master`.

In general, I like that you introduced a `getListMetricsInput` func. I think we should take this opportunity to:
- make a `GetListMetricsInput` exported func
- change stuff in `pkg/job` to call it
- change the signature of `ListMetrics` to take `ctx` and `input`

Reply:
@PerGon do you have time to pick this up again?

Reply:
I've rebased and if I understood correctly, I got your suggestions in.

How does it look now?

Reply:
@PerGon the change looks good, but after merging https://github.com/nerdswords/yet-another-cloudwatch-exporter/pull/929 we're going in a different direction. I think you can basically go back to what was your original implementation, with the new param passed to `ListMetrics`. Sorry for the noise.

Reply:
No worries. Reverted back to the initial implementation - rebased to master.

How do you suggest to proceed?